.github/workflows/linters.yml

```diff
@@ -25,22 +26,12 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
-      - name: Get yarn cache directory path
```

Comment:
Why remove this?

Reply:
Because the setup-node action supports caching built in, so we can remove all this